client/my-sites/comments/style.scss

```diff
@@ -68,6 +68,11 @@
 	}
 }
 
+.comment-navigation__sort-buttons {
+	margin-right: 16px;
+	float: left;
```

Comment:
Were we unable to use flexbox?

Reply:
No, we can use flexbox too to achieve the same result. Any particular reason to prefer one over the other in this case?

Reply:
If you change to flex, please check if `rtl` works as expected.
AFAIK, floats are reversed automagically, while I'm not sure it happens for flex as well.

Reply:
Clearing floats tends to be a pain, so if we're able to use flexbox I usually prefer that technique.